_config/mailcapture.yml

```diff
@@ -6,8 +6,8 @@ Injector:
     class: CaptureMailer
 #    properties:
 #      outboundMailer: %$MailerObject
-#      #Set to false to enable pass through of emails without logging
-#      captureEmails: TRUE
+#      # Set to false to enable pass through of emails without logging
+#      captureEmails: FALSE
```

Comment:
This YML is commented out.. but the comment says "Set to false to enable pass through of emails without logging"

So... shouldn't it be set to FALSE?
(Though I'm now realizing this is probably here for a quick-copy paste reason so if you want me to revert, will do)